server/coordinator/procedure/dispatch/dispatch.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+package dispatch
+
+import (
+	"github.com/CeresDB/ceresdbproto/pkg/clusterpb"
+)
+
+type EventDispatch interface {
+	SendOpenEvent(ShardIDs []uint32, targetNode string) (bool, error)
```

Comment:
It's confusing to return a boolean value, any comments could explain it or just remove it.